.github/workflows/kuzzle-runner.workflow.yml

```diff
@@ -0,0 +1,101 @@
+name: kuzzleio/kuzzle-runner
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm/v7"
+  NODE_LTS_MAINTENANCE_VERSION: "16"
+  NODE_LTS_ACTIVE_VERSION: "18"
+  NODE_LTS_CURRENT_VERSION: "20"
```

Comment:
Why not use a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) for doing the same thing?